netsim/ansible/templates/initial/cumulus.j2

```diff
@@ -135,10 +136,6 @@ interface {{ l.ifname }}
 {% elif l.type|default("") == "stub" %}
  description Stub interface
 {% endif %}
-{% if 'ipv6' not in l.dhcp.client|default({}) %}
```

Comment:
Without 'ipv6 nd ra' the EBGP LLA sessions won't start. It must be enabled.

Reply:
Could we at least make it conditional on ```if l.ipv6|default(False)```?

Reply:
No. That would break the "unnumbered IPv4 EBGP sessions" that really rely on IPv6 LLA behind the scenes.

Why is it bothering you?

Reply:
I'm working on an ipv4-only network, and there is no ipv6 anywhere in the configs. 

There is no BGP either, so if this setting is intended to fix an issue with BGP, perhaps we could do 
```if 'bgp' in modules { ...``` ?

Reply:
Added ```if 'bgp' in module``` condition

Reply:
It looks like you're throwing spaghetti at the wall. I learned the hard way that doesn't end well.

If you have a problem with that particular bit of template, please open a separate issue, explain why it bothers you, and we might decide to fix it. Changing some random stuff in an unrelated PR usually results in hard-to-figure-out problems further down the line.

Reply:
ok, left the comment explaining why it's there but restored the condition to the way it was